Makefile

```
@@ -224,22 +271,46 @@
+else
 	$(DOCKER_COMPOSE) exec dev sh
+endif
 
 ps: ## Log to the docker container
+ifeq ($(DIND), 1)
+	@$(DOCKER_COMPOSE) $(COMPOSE_FILES) ps
+else
 	@$(DOCKER_COMPOSE) ps
+endif
 
 logs: ## Show all logs
```

checkmake (0.2.2)

[warning] Target "logs" should be declared PHONY.

(phonydeclared)